<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNetDialog</class>
 <widget class="QDialog" name="AddNetDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Bus</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
border: 0px transparent;
max-width:120px;
max-height: 40px;
min-width:120px;
min-height: 40px;
color: rgb(255,255,255);
font: 12px &quot;Lucida Sans&quot;;
}

#okButton {
border-image: url(:/core/small-blue-but-normal.png);
}

#cancelButton {
border-image: url(:/core/small-gray-but-normal.png);
}

#rescanButton {

border-image: url(:/core/small-blue-but-normal.png);
border: 0px transparent;
max-width:75px;
max-height: 24px;
min-width:75px;
min-height: 24px;
color: rgb(255,255,255);
font: 12px &quot;Lucida Sans&quot;;
margin:5px;
}


QComboBox:drop-down {
border-image: url(:/core/drop-down-sm-blue.png);
border: 0px transparent;
}

#AddNetDialog {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(253, 253, 253, 255), stop:1 rgba(208, 208, 208, 255));
}</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0">
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="1" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>COM Port</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="rescanButton">
       <property name="text">
        <string>Rescan</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="5">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QLabel" name="portHintLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>90</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Bus Name</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="4">
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="cancelButton">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="okButton">
          <property name="text">
           <string>Add Bus</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QComboBox" name="portCombo"/>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="netName"/>
     </item>
     <item row="2" column="1">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
